### PR TITLE
[EPMDEPS-15617] Preparing for Labeling tool packaging_2

### PR DESCRIPTION
**Pull request by NatalliaHutovich**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ export default App
 3. Update `yourConfigHere` with your desired configurations.
 
 ## Demo
-There is a demo hosted on [Demo Link] which allows to check _labeling-tool_ package.
+There is a demo hosted on https://epam.github.io/deps-fe-labeling-tool which allows to check _labeling-tool_ package.
```
Comment (Rob-n-Dude) [inline README.md:153]: should we update the patch version?

Comment (Rob-n-Dude) [reply README.md:153]: if it is already deployed -> then yes, we have to update version + redeploy.

if no -> no need to update version

File: package.json [hunk 2]
```diff
@@ -19,7 +19,9 @@
     "test:ci": "node --expose-gc --trace-warnings --unhandled-rejections=strict ./node_modules/jest/bin/jest.js --bail --ci --colors --expand --errorOnDeprecated --coverage --reporters=default --maxWorkers=1 --forceExit --logHeapUsage",
     "test:manual": "jest --expand --colors --watch",
     "test:debug": "node --inspect-brk ./node_modules/jest/bin/jest.js --expand --colors --watch --runInBand",
-    "prepublishOnly": "node scripts/npm-package.js"
+    "prepublishOnly": "node scripts/npm-package.js",
+    "deploy": "gh-pages -d dist",
+    "predeploy": "npm run build:prod"
```
Comment (DrDiman) [inline package.json:24]: let's have the only one script `deploy` it will do all combined magic.

File: README.md (hunk 1 above)
Comment (DrDiman) [inline README.md:153]: let's add formatting?
```
[GitHub gh-pages](https://epam.github.io/deps-fe-labeling-tool).
```

**Review by DrDiman — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (NatalliaHutovich) [reply README.md:153]: No, package was not deployed yet

Comment (NatalliaHutovich) [reply README.md:153]: May be text [Demo Labeling Tool] ?

File: package.json (hunk 2 above)
Comment (NatalliaHutovich) [reply package.json]: Removed predeploy script

**Comment by Rob-n-Dude** [comment]
LGTM 👍

**Pull request merged**